src/SystemLoader_TEST.cc

```diff
@@ -43,7 +43,7 @@ TEST(SystemLoader, Constructor)
       "<world name='default'>"
       "<plugin filename='libignition-gazebo") +
       IGNITION_GAZEBO_MAJOR_VERSION_STR + "-physics-system.so' "
-      "name='ignition::gazebo::systems::Physics'></plugin>"
+      "name='gazebo::systems::Physics'></plugin>"
```

Comment:
Within string

Reply:
Oh.. I gotta catch `'` rather than just `"` too, oops.
I'll revert it tomorrow

Reply:
https://github.com/gazebosim/gz-sim/compare/355b08bcddceeabb3ca91481e04feef2e7b98f69..5529d93c595010177c80f1cde37bab6bdbdc5337

The components::Name is from a different escape hatch targeting another part of `ignition-gazebo`... I'm surprised it compiled properly the first time